api/v1beta1/vmagent_types.go

```diff
@@ -416,7 +416,7 @@ type VMAgentRemoteWriteSettings struct {
 	// +optional
 	// +kubebuilder:validation:Pattern:="[0-9]+(ms|s|m|h)"
 	FlushInterval *string `json:"flushInterval,omitempty"`
-	// Optional labels in the form 'name=value' to add to all the metrics before sending them
+	// Labels in the form 'name=value' to add to all the metrics before sending them. This overrides the label if it already exists.
```

Comment:
I suppose that this comment (about `'name=value'`) may be taken the wrong way, because in the custom resource yaml it looks like following:

```yaml
# ...
labels:
  name: value
````

Also after changing the API you need to run 'make build' to generate a new CRD bundle.

Reply:
Thanks! Updated